telethon/client/uploads.py

```diff
@@ -286,6 +287,15 @@ async def send_file(
                 Not all types of media can be used with this parameter, such
                 as text documents, which will fail with ``TtlMediaInvalidError``.
 
+            nosound_video (`bool`. optional):
+                This value only applies if sending a video without an audio
+                track. If True, the video will be displayed in telegram as a
```

Comment:
Should reuse the fixed documentation from the other method.

Reply:
Copied over